packages/react-core/src/hooks/index.ts

```diff
@@ -2,6 +2,7 @@ export {
   default as useDeprecationWarning,
   UseDeprecationWarning,
 } from './useDeprecationWarning';
+export * from './useGetUrl';
```

Comment:
Please follow existing patterns and only export what is needed, export all syntax shorthand can lead to unintentional exports